src/app/pre-login/page.tsx

```diff
@@ -0,0 +1,10 @@
+"use client";
```

Comment:
Can we remove this? I don't think we need this here, only inside FastLoginPage. 

Can you confirm @pedrohbl03?